<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commodityinfoinput</class>
 <widget class="QWidget" name="commodityinfoinput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>超市商品管理系统-商品信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/img1/img/winIcon.jpg</normaloff>:/img1/img/winIcon.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="type_box">
     <item>
      <property name="text">
       <string>食品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>饮料</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>生活用品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>化妆品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="brand_edit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="producer_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>商品名称：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="name_edit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="price_edit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>商品厂家:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QWidget" name="widget_btn" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="cancel_btn">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="ok_btn">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>商品品牌:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>商品价格:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>商品种类:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="num_edit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>商品数量:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>type_box</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>price_edit</tabstop>
  <tabstop>brand_edit</tabstop>
  <tabstop>producer_edit</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
